int32  		pos
int16  		duty
uint16  	current
int16  		freq
uint16 		mot_state

uint16 MotorStop = 0
uint16 MotorRun  = 1
uint16 MotorErr  = 2